test/policy.bats

```diff
@@ -204,3 +204,38 @@ SANDBOX_CONFIG="$TESTDATA/sandbox_config.json"
 
 	[[ "$output" == *"SignatureValidationFailed"* ]]
 }
+
+@test "allow signed image with restrictive policy on container creation7 if already pulled (by tag and ID)" {
```

Comment:
This is fundamentally confusing things. “Image ID” (`StorageImageID`) is, historically (nowadays not always) the sha256 digest of the config, and usually expressed without the leading `sha256:`.

“image digest” is the sha256 of the manifest. The two are ~never the same for a single image.

In `busybox:latest@sha256:…`, the second value is the manifest digest. So assuming the value from `crictl images -q` is an image ID, using it in the manifest digest syntax should never find a match; and the test name is misleading.

(Also for the other added test.)